README.md

```diff
@@ -276,6 +276,13 @@ To run a Task that processes CSVs from the command line, use the --csv option:
 $ bundle exec maintenance_tasks perform Maintenance::ImportPostsTask --csv 'path/to/my_csv.csv'
 ```
 
+To run a Task that takes arguments from the command line, use the --arguments
+option, passing arguments as a set of <key>:<value> pairs:
+
+```bash
+$ bundle exec maintenance_tasks perform Maintenance::ParamsTask --arguments post_ids:1,2,3 content:"Hello, World!"
```

Comment:
I'd love if the example Task in the README match the CLI call here and the task in the dummy app, but I can look into this later, it doesn't need to be in this PR.